pkgs/data/fonts/ttf-bitstream-vera/default.nix

```diff
@@ -1,9 +1,9 @@
-{stdenv, fetchzip}:
+{ stdenv, fetchzip, gnome3 }:
 
-fetchzip {
+fetchzip rec {
   name = "ttf-bitstream-vera-1.10";
 
-  url = mirror://gnome/sources/ttf-bitstream-vera/1.10/ttf-bitstream-vera-1.10.tar.bz2;
+  url = "mirror://gnome/sources/ttf-bitstream-vera/${stdenv.lib.versions.majorMinor version}/${name}.tar.bz2";
```

Comment:
Eval error here because `version` does not exist.

You'll need to use `pname` and `version` above.

Reply:
yeah, i've got a fix on my local tree.